docs/src/pages/components/link/demo.tsx

```diff
@@ -14,21 +27,23 @@ export const LinkDemo = () => {
   });
 
   return (
-    <View>
-      <LinkPropControls {...linkProps} />
-      <Example>
+    <Demo
+      code={propsToCode(linkProps)}
+      propControls={<LinkPropControls {...linkProps} />}
+    >
+      <>
         <Link
           color={linkProps.color}
           isExternal={linkProps.isExternal}
           textDecoration={linkProps.textDecoration}
         >
-          {linkProps.children}
+          My Demo Link
         </Link>
-        <Text fontSize={'.75em'}>
+        <Text fontSize="0.75em">
```

Comment:
can we use a design token here?
`const { tokens } = useTheme();`
`<Text fontSize={tokens.space.relative.small}>`